Products/ZenModel/migrate/updateHBaseForNetcat.py

```diff
@@ -0,0 +1,67 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.0.0")
+
+
+class UpdateHBaseForNetcat(Migrate.Step):
+    """ Recursively update all the HBase service healthchecks.  Necessary
+    due to the hbase image changing from an ubuntu base to a centos base,
+    which means BSD netcat to GNU netcat """
+
+    version = Migrate.Version(5,1,70)
```

Comment:
Looks like we're inconsistent on these versions across the service migration scripts. Most are 5,0,70, but some are 5,1,70. Not sure what the right number should be, but seems like all of the service migrations for 5.1.1 should have the same version